runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -8,4 +8,9 @@ metadata:
     - "web"
     - "reactive"
   status: "experimental"
+  codestart:
+    name: "quarkus-hilla-maven-starter"
+    languages:
+      - "java"
+    artifact: "com.github.mcollovati:quarkus-hilla:codestarts:jar:${project.version}"
```

Comment:
codestarts as singular or plural?  Or name it starter directly. Don't know how it is named in other projects.

Reply:
It should have that exact classifier, if I got correctly the documentation

Reply:
The classifier is taken from the documentation: https://quarkus.io/guides/extension-codestart#the-codestart-quarkiverse-or-standalone-extensions